spk/wallabag/src/parameters.yml

```diff
@@ -1,34 +1,64 @@
 parameters:
     database_driver: pdo_mysql
+    database_driver_class: ~
     database_host: 127.0.0.1
     database_port: 3306
```

Comment:
Bad luck. On my test system, I have installed both MariaDB 5 and 10... so port is 3307 now